src/web/routes/register-routes.js

```diff
@@ -21,11 +24,17 @@ const registerRoutes = (config, app) => {
 
   app.use(setCommonTemplateValues)
 
-  registerFormRoutes(csrfProtection, steps, app)
-  registerStartRoute(app)
-  registerCheckRoutes(csrfProtection, config, app)
-  registerConfirmRoute(app)
-  registerCookiesRoute(app)
+  const maintenanceMode = getVCAPServicesVariable('MAINTENANCE_MODE', false)
```

Comment:
Is it possible to set `maintenanceMode` on the `config` object that is passed down from app initialisation? `config` is already available inside `registerRoutes` .. it's the approach that we've taken with the rest of the app.

Reply:
Moved into config.